### PR TITLE
Search rpmbuild in more paths

### PR DESCRIPTION
**Pull request by xrmx**
So there's more chances that is found. At least on MacOS when
installed with brew. This is the minimal fix i can find to help
with: https://github.com/pypa/setuptools/issues/1456.

### CODE REVIEW

File: Lib/distutils/command/bdist_rpm.py [hunk 1]
```diff
@@ -311,6 +311,7 @@ def run(self):
         log.info("building RPMs")
         rpm_cmd = ['rpm']
         if os.path.exists('/usr/bin/rpmbuild') or \
+           os.path.exists('/usr/local/bin/rpmbuild') or \
            os.path.exists('/bin/rpmbuild'):
             rpm_cmd = ['rpmbuild']
```
Comment (asottile) [inline Lib/distutils/command/bdist_rpm.py:314]: maybe even use `shutil.which` instead?

Comment (xrmx) [reply Lib/distutils/command/bdist_rpm.py:314]: makes sense, updated the PR, thanks!

**Comment by xrmx** [comment]
I've rebased the PR to latest master. I consider this patch trivial so I've omitted the issue and the news file, are something else from my side missing to move this PR forward? Thanks

**Review by asottile — APPROVED** [review]

**Comment by csabella** [comment]
Closing as duplicate of #10594.

**Pull request closed without merging**